tests/integration/test_frozenarray.py

```diff
@@ -16,10 +16,10 @@ def main() -> int:
     validate(compiled)
     # TODO: Enable execution test once LLVM lowering is done:
     #  https://github.com/CQCL/hugr/issues/1973
-    #run_int_fn(compiled, 42)
+    # emulate_int_fn(compiled, 42)
```

Comment:
This is done now, so we could enable the tests. Happy for that to be a different PR though